Scripts/script-ReadPDFFile.yml

```
@@ -29,9 +50,18 @@ script: |-
   else:
       path = demisto.get(res[0],'Contents.path')
       if path:
-          pdf = pdfx.PDFx(path)
+          try:
+              pdf = pdfx.PDFx(path)
+          except PSEOF:
+              mark()
```

Same as above comment
```suggestion
              mark_suspicious()
```